MMBWMON messages

jbreitbart opened the pull request:
These are the messages currently used by the main memory bandwidth monitoring agent.

spickartz commented:
An update to the "Struktur" part is missing. And we should discuss about standardizing our naming scheme. We have "task/result", "request/response", and now "request/reply". This will probably result in unnecessary confusion, what do you think?

jbreitbart commented:
Added mmbwmon to structure.

Updated reply -> response, but I really think we have more important matters at hand than a unified naming scheme for prototypes :wink:
